content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -9,6 +9,21 @@ desc: "Changelog of the Agent Chat API"
 
 This document is the record of all the changes in the **Agent Chat API**, Web and RTM references, starting from **version 3.0**.
 
+## [v3.3]
+
+### Changed
+
+Many methods had their parameters renamed:
+- `deactivate_chat` - `chat_id` => `id`
+- `follow_chat` - `chat_id` => `id`
+- `unfollow_chat` - `chat_id` => `id`
+- `transfer_chat` - `chat_id` => `id`
+- `update_chat_properties` - `chat_id` => `id`
+- `delete_chat_properties` - `chat_id` => `id`
+- `get_customer` - `customer_id` => `id`
+- `update_customer`  - `customer_id` => `id`
+- `ban_customer`  - `customer_id` => `id`
+
```

Comment:
Can we keep changelog similar as in other PR: https://github.com/livechat/livechat-public-docs/pull/713

Reply:
@kogierman, could you adopt the new changelog style, please? See v3.2 changelog for reference (it's already on prod).
Here, I'd do the following entries:

> ### Chats
> In the **Deactivate Chat** ([Web] & [RTM]) method, the `chat_id` parameter was renamed to `id`.
> 
> ### Properties
> In the **Update Chat Proeprties** ([Web] & [RTM]) method, the `chat_id` parameter was renamed to `id`.
> 

...and so on.

If there's more than 1 change for a method, you can do a nested list, for example:

> ### Chats
> 
> The **Deactivate Chat** ([Web] & [RTM]) method:
>  - had the `chat_id` parameter renamed to `id`.

Reply:
sure, I'll align to new format